pkg/compute/startup.go

```diff
@@ -53,23 +51,21 @@ func (s *Startup) ensureLiveJobs(ctx context.Context) error {
 	for idx := range localExecStates {
 		localExecution := localExecStates[idx]
 
-		switch localExecution.Execution.Job.Type {
-		case models.JobTypeService, models.JobTypeDaemon:
-			{
-				// Service and System jobs are long running jobs and so we need to make sure it is running
-				err = s.runExecution(ctx, localExecution)
-				if err != nil {
-					errs = multierror.Append(errs, err)
-				}
+		// Fetch the restart policy for the current task.
+		policy := localExecution.Execution.Job.Task().RestartPolicy
+
+		if policy.Attempts > 0 {
```

Comment:
`models.Task` should not be updated by the system and should only be updated by the user. You shouldn't decrement `Attempts`, and instead increment something in the `localExecutionState` or in the `models.Execution`, and compare that with task attempts

Reply:
Ah yeah, that's what I was trying to get at this morning with my questions around copying stuff from tasks into executions.